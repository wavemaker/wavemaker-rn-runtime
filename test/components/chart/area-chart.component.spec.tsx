import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmAreaChart from '@wavemaker/app-rn-runtime/components/chart/area-chart/area-chart.component';

describe('Test AreaChart component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmAreaChart name="test_AreaChart"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});
