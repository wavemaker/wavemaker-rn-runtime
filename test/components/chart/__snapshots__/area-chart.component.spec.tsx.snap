// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test AreaChart component Should render component 1`] = `
<View
  accessible={true}
  onLayout={[Function]}
  style={{}}
>
  <View>
    <View
      style={
        {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <Text
        style={
          {
            "color": "#333333",
            "fontSize": 20,
            "lineHeight": 24,
            "paddingLeft": 10,
          }
        }
      />
    </View>
    <Text
      style={
        {
          "color": "#666666",
          "fontSize": 12,
          "lineHeight": 16,
        }
      }
    />
  </View>
</View>
`;
